pypesto/objective/petab_import.py

```diff
@@ -207,19 +207,32 @@ def create_edatas(self, model=None, simulation_conditions=None):
 
             # add measurements and sigmas
             # iterate over time points
-            for time_ix, time in enumerate(timepoints_w_reps):
+            for time in timepoints:
                 # subselect for time
                 df_for_time = df_for_condition[df_for_condition.time == time]
+                time_ix_0 = timepoints_w_reps.index(time)
+
+                # remember used time indices for each observable
+                time_ix_for_obs_ix = {}
+
                 # iterate over measurements
                 for _, measurement in df_for_time.iterrows():
-                    # get observable index
+                    # extract observable index
                     observable_ix = observable_ids.index(
                         f'observable_{measurement.observableId}')
```

Comment:
I'm really annoyed by all those `f'observable_{...}'`in the code and the ambiguity of observable id... :expressionless:

Reply:
you mean that it's ``observable_`` in the sbml, but not in the measurement file? yeah, kind of not optimal, and it would be good to have a unique id used everywhere.
but it's good to have the id in the sbml to directly see what's an observable, and have it not in the measurement_df, because there we really don't need it as it does not help readability. so this solution also makes sense ...

Reply:
Yes. And that we call both observable_id, which regularly confuses me. General issue, not specific to this PR.